docs/tutorials/rootless_tutorial.md

```diff
@@ -142,16 +106,16 @@ The three main configuration files are [containers.conf](https://github.com/cont
 Podman reads
 1. `/usr/share/containers/containers.conf`
 2. `/etc/containers/containers.conf`
-3. `$HOME/.config/containers/containers.conf`
+3. `${XDG_CONFIG_HOME}/containers.conf`
```

Comment:
1. path is of, it must be `${XDG_CONFIG_HOME}/containers/containers.conf`
 2. XDG_CONFIG_HOME is not set by default so it should say that it uses $HOME/.config if it is unset IMO.